lib/ZumoOta/src/Upload.cpp

```diff
@@ -98,14 +101,36 @@ void Upload::handleFileUpload(AsyncWebServerRequest *request, const String& file
     if (final)
     {
         request->_tempFile.close();
-        /*Check if the file exists in the file system*/
-        if(LittleFS.exists(updatedFilename))
+       /* Check if the file exits in FileSystem */
+        if (LittleFS.exists(updatedFilename))
         {
-            LOG_DEBUG(String(updatedFilename) + " " + "exists in FileSystem.");
+            /* Open the file in read mode */
+            File file = LittleFS.open(updatedFilename, "r");
+
+            if (file)
+            {
+                /* Move to the end of the file */
+                file.seek(0, SeekEnd);
+
+                /* Get the current position in the file content (the size of the file) */
+                size_t fileSize = file.position();
+
+                /* Move back to the beginning of the file */
+                file.seek(0, SeekSet);
+
+                LOG_DEBUG("Size of " + updatedFilename + ": " + String(fileSize));

```

Comment:
It is not only about printing size, but also to verify it with the size provided by web browser . see this erlier shared link: https://github.com/search?q=repo%3ABlueAndi%2Fesp-rgb-led-matrix%20X-File-Size-Firmware&type=code

Reply:
It will be performed by the next commit.